transports/msmq/index.md

```diff
@@ -71,12 +71,14 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
+| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+
+NOTE: If you are running in *Workgroup* mode then Windows will grant Send permissions to Anonymous. Everyone is added by default by Windows but not with Send or Receive permisssions. Prior to NServiceBus 6.1.0 Send permissions for Anonymous and Everyone were granted at queue creation but this behavior was altered to not accidentally open up permissions.
```

Comment:
How's this:

When an NServiceBus endpoint creates a queue on a machine, the default permissions depend on whether the machine is connected to a domain or a workgroup. 

If the machine is connected to a domain, then only the domain user running the endpoint when the queue is created will have `Send` permissions granted. If two endpoints need to communicate and are running under the same domain account, no further configuration is required. If two endpoints need to communicate using different domain accounts then the `Send` permission on the receiving endpoint input queue needs to be granted to the domain account of the sending endpoint.

For endpoints using the publish-subscribe pattern, both the publisher and the subscriber must have permission to send messages to each other. This is to allow the `Subscribe` message to be sent from the subscribing endpoint to the publishing endpoint.

If the machine is connected to a workgroup then the `Send` permission is granted to the `Everyone` and `Anonymous` user groups by Windows. Any endpoint will be able to send messages to any other endpoint without further configuration. 

----

After this you need the warning. I would maybe even just link off to the security advisory. It's not just new queues either. If you have run an older endpoint in prod, it may have granted additional permissions to an existing queue.